files/common/var/lib/delphix-platform/os-migration

```diff
@@ -84,6 +84,28 @@ function perform_migration() {
 	chown -R postgres:postgres /mds ||
 		die "Failed chowning /mds"
 
+	#
+	# For each domain0 dataset with a default mountpoint,
+	# the parents need other-execute for NFSv4 exports.
+	#
+	DATASETS=$(zfs list -H -r -t filesystem -o mountpoint domain0 |
+		grep ^/domain0/ | xargs dirname | sort | uniq)
+
+	for DATASET in $DATASETS; do
+		if [[ "$DATASET" == "/domain0" ]]; then
+			continue
+		fi
+		PERMS=$(stat "$DATASET" -c %a 2>/dev/null)
+		RETURN=$?
+		if [[ $RETURN -eq 0 && "$PERMS" == "750" ]]; then
+			echo "Fixing execute permissions: $DATASET" \
```

Comment:
Since we are already `echo`ing into the service log on the line below, we can probably remove this.

Reply:
Good point.  Removing.